packages/gestalt/src/Touchable.js

```diff
@@ -73,126 +82,167 @@ const getRoundingStyle = (r: Rounding): Style => {
+  accessibilityExpanded,
+  accessibilityHaspopup,
+  accessibilityLabel,
+  children,
+  disabled = false,
+  fullWidth = true,
+  fullHeight,
+  mouseCursor = 'pointer',
+  onBlur,
+  onFocus,
+  onMouseEnter,
+  onMouseLeave,
+  onTouch,
+  pressBackgroundEnabled = false,
+  pressScaleEnabled = false,
+  rounding = 0,
+}: Props) {
+  const [isFocused, setFocused] = React.useState<boolean>(false);
+  const [isPressed, setPressed] = React.useState<boolean>(false);
+  const [coordinate, setCoordinate] = React.useState<Coordinate>({
+    x: 0,
+    y: 0,
+  });
+  const ignoreNextFocusEventRef = React.useRef<boolean>(false);
+
+  const classes = classnames(
+    styles.touchable,
+    toProps(getRoundingStyle(rounding)).className,
+    {
+      [styles.fullHeight]: fullHeight,
+      [styles.fullWidth]: fullWidth,
+      [styles[mouseCursor]]: !disabled,
+      [styles.pressScale]: !disabled && pressScaleEnabled && isPressed,
+      [styles.pressBackground]:
+        !disabled && pressBackgroundEnabled && (isPressed || isFocused),
     }
-  };
-
-  handleMouseLeave = (event: SyntheticMouseEvent<HTMLDivElement>) => {
-    const { disabled, onMouseLeave } = this.props;
-    if (!disabled && onMouseLeave) {
-      onMouseLeave({ event });
-    }
-  };
-
-  handleClick = (event: SyntheticMouseEvent<HTMLDivElement>) => {
-    const { disabled, onTouch } = this.props;
-    if (!disabled && onTouch) {
-      onTouch({ event });
-    }
-  };
-
-  render() {
-    const {
-      accessibilityControls,
-      accessibilityExpanded,
-      accessibilityHaspopup,
-      accessibilityLabel,
-      children,
-      disabled = false,
-      fullWidth = true,
-      fullHeight,
-      mouseCursor = 'pointer',
-      rounding = 0,
-    } = this.props;
-
-    const classes = classnames(
-      styles.touchable,
-      toProps(getRoundingStyle(rounding)).className,
-      {
-        [styles.fullHeight]: fullHeight,
-        [styles.fullWidth]: fullWidth,
-        [styles[mouseCursor]]: !disabled,
-      }
-    );
-
-    return (
-      <div
-        aria-controls={accessibilityControls}
-        aria-disabled={disabled}
-        aria-expanded={accessibilityExpanded}
-        aria-haspopup={accessibilityHaspopup}
-        aria-label={accessibilityLabel}
-        className={classes}
-        onClick={this.handleClick}
-        onBlur={this.handleBlur}
-        onFocus={this.handleFocus}
-        onMouseEnter={this.handleMouseEnter}
-        onMouseLeave={this.handleMouseLeave}
-        onKeyPress={this.handleKeyPress}
-        role="button"
-        tabIndex={disabled ? null : '0'}
-      >
-        {children}
-      </div>
-    );
-  }
+  );
+
+  return (
+    <div
+      aria-controls={accessibilityControls}
+      aria-disabled={disabled}
+      aria-expanded={accessibilityExpanded}
+      aria-haspopup={accessibilityHaspopup}
+      aria-label={accessibilityLabel}
+      className={classes}
+      onContextMenu={event => event.preventDefault()}
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event

This has always been set in Duplo. See https://phabricator.pinadmin.com/D192884